views/beneficial-owner-type.html

```diff
@@ -10,55 +10,17 @@
 
 {% block content %}
 
-  <div class="govuk-grid-row govuk-body">
+  <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
-      <form method="post">
+      <form>
 
-        <h1 class="govuk-heading-xl govuk-!-margin-0">Who is a beneficial owner of the overseas entity?</h1>
-        <br/>
-        {% set ownerHint %}
-          {{ govukDetails({
-            summaryText: "What is a beneficial owner?",
-            text: "A beneficial owner is an individual who ultimately owns or controls more than 25% of a company's shares or voting rights, or who otherwise exercise control over the company or its management."
-          }) }}
-          <p class="govuk-hint">Select all that apply.</p>
-        {% endset %}
+        {% if (statement.beneficialOwnerStatement === "allIdentifiedAllSupplied") %}
+          {% include "includes/beneficial-owners-check.html" %}
+        {% else %}
+          {% include "includes/beneficial-owners-check.html" %}
```

Comment:
the include for the 'else' is the same as the 'if' - could just remove the 'if else' and just include

Reply:
A non issue as this is only temporary. I want the if else structure there as this is part of the development to match the AC but also want to ensure that we still get something on the page. This will be addressed on the next ticket.